lib/shellable.ts

```diff
@@ -226,7 +234,7 @@ export class Shellable extends cdk.Construct {
     }
 
     this.buildSpec = BuildSpec.simple({
-      preBuild: this.platform.prebuildCommands(props.assumeRole),
+      preBuild: this.platform.prebuildCommands(props.assumeRole, props.useRegionalSTSEndpoints),
```

Comment:
Why is this not just an environment variable on the CodeBuild project?

Reply:
Could you please explain in more detail what you mean?
If you mean, that clients that use `aws-delivlib` should set this environment variable by themselves, then I can see 2 problems here:
1. They might just forget to set it.
2. They might not know that `aws-delivlib` makes a call `aws sts assume-role`, so they will not set that variable.

So having it inside of `aws-delivlib` makes it easier to use and less error-prone.

Reply:
Did you mean adding to [ShellableOptions.environment](https://github.com/awslabs/aws-delivlib/blob/master/lib/shellable.ts#L35) `{ AWS_STS_REGIONAL_ENDPOINTS: regional }`, so it will be used in [cbbuild.Project](https://github.com/awslabs/aws-delivlib/blob/master/lib/shellable.ts#L245) ?

Would it be equivalent to setting this environment variable just before the call assume role in `AWS_STS_REGIONAL_ENDPOINTS=regional aws sts assume-role`?

Reply:
It wouldn't be entirely equivalent. The reach would be wider, because it would apply to ALL commands. But that seems like something you'd want.

Reply:
I agree that it's probably something most users would want, but maybe not all.
This small change gives a nice scoped control over this variable and seems to be quite innocent.